xbmc/video/VideoDatabase.cpp

```diff
@@ -10282,3 +10516,34 @@ bool CVideoDatabase::SetVideoUserRating(int dbId, int rating, const MediaType& m
   }
   return false;
 }
+
+void CVideoDatabase::InitializeTypeTable()
+{
+  const char* types[] = { "Standard",
+                          "Extended",
+                          "Unrated",
+                          "Uncut",
+                          "Remastered",
+                          "Limited",
+                          "Special",
+                          "Director Cut",
+                          "The Final Cut",
+                          "Super Duper Cut",
+                          "Theatrical",
+                          "3D",
+                          "4K UHD",
+                          "IMAX",
+                          "Collector",
+                          "Ultimate Collector",
+                          "Criterion Collection",
+                          "10th Anniversary",
+                          "20th Anniversary",
+                          "25th Anniversary",
+                          "30th Anniversary",
+                          "40th Anniversary",
+                          "50th Anniversary",
+  };
```

Comment:
This is just try to eliminate the work for user to enter them, input them with remote is not very convenient.

Reply:
I think there's value in predefined versions, but they definitely shouldn't be declared on line 10,540

Reply:
I won't decide if those values are wanted, but if they are they should be defined somewhere else.
I haven't looked yet what a better place would be.

(@xodidox please don't mark this as resolved yet, so that others can see this comments)

Reply:
They should be translatable to some degree or even user defined.

Reply:
I considered add these names to the language file, so they can be localized. But if it's a new installation of Kodi, the table is created before user can choose the UI language, there is no way to use the localized version of these names. 

User can always define these values and/or any new value, I added them mainly because these common names will cover most users' needs and they can just use them with a click. Input them with a remote is not very convenient.

Reply:
Does version need to be codified at all? What I mean is could this feature be implemented without having a lookup table of version values, forming the list of different movie version values by looking at the movie table. I know that means doing things differently from Genre and Tag, but could be worth considering.

As for localization of the version names, if you do continue with a lookup table of values  consider populating it spearately from table creation at a time when UI language is available. Although what to do if the user changes language?